### ISSUE TITLE
Why Minimizing the Output of the Residual Network?

### ISSUE THREAD

**Issue by WillCheung2016** [ctom2/colie#1]
Hi,

Thanks for open-sourcing the code. I notice a problem in the paper and cannot understand. Please enlighten me.

At this [line](https://github.com/ctom2/colie/blob/77fec3327bcfa48e29a9c41659f4a0874c2deaf4/colie.py#L61), illu_lr = illu_res_lr + img_v_lr. When computing the fidelity loss at this [line](https://github.com/ctom2/colie/blob/77fec3327bcfa48e29a9c41659f4a0874c2deaf4/colie.py#L65), the term illu_lr - img_v_lr simply equals illu_res_lr. Therefore, the fidelity loss simply minimizes the output of the network. This term seems to be more like a regularization loss than a supervised loss as it doesn't use any label information. I think the fidelity loss is supposed to be a supervised loss using model outputs and the V component of the image. Could you please correct me if my understanding is wrong?

**Comment by ctom2** [comment]
Thank you for your interest in our work. Our model does not rely on supervised label information. Instead it is trained directly using the input image and by leveraging solely the image's inherent information through zero-shot optimization guided by assumptions from Retinex theory.

Therefore, for each input image the weights of the network need to be updated to fit the needs of the specific image.

Please see more details about the loss function composition and the details [here](https://arxiv.org/abs/2407.12511).

**Comment by WillCheung2016** [comment]
Thanks for replying. As far as I understand the paper with the code at this [line](https://github.com/ctom2/colie/blob/77fec3327bcfa48e29a9c41659f4a0874c2deaf4/colie.py#L65), this mean square loss should use the V component of the HSV image **img_v_lr** as the supervision signal.

loss_spa = torch.mean(torch.abs(torch.pow(illu_lr - img_v_lr, 2)))

My question is, if you define illu_lr = illu_res_lr + img_v_lr, then the supervision signals in the mean square loss function would be nullified, which is equivalent to minimizing the network's output illu_res_lr.
